application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/wolframalpha/Tips.java

```diff
@@ -0,0 +1,49 @@
+package org.togetherjava.tjbot.commands.mathcommands.wolframalpha;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlElementWrapper;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * <p>
+ * Example Query: bhefjuhkynmbtg <br>
+ * Result: <br>
+ * {@code <tips count='1'>
+ *   <tip text='Check your spelling, and use English' />
+ *  </tips>}
+ * </p>
+ */
+@SuppressWarnings("unused")
```

Comment:
either
* use the class,
* remove the class,
* or remove the suppression.

Reply:
I have suppressions for `QueryResult` too because the linter does not understand it is needed for the jackson library